lib/admin/components/ServerSettings.js

```diff
@@ -222,6 +222,9 @@ class ServerSettings extends Component<Props, State> {
                   index={index}
                   fields={SERVER_FIELDS}
                   defaultExpanded={!server.name}
+                  testId={server.name
```

Comment:
The `||` notation should work here, correct?

Reply:
yep